3.6/Dockerfile.rhel7.tbd

```diff
@@ -0,0 +1,74 @@
+# This image provides a Python 3.6 environment you can use to run your Python
```

Comment:
As @pkubatrh mentioned, it would be better to [update container-common-scripts](https://github.com/sclorg/container-common-scripts/pull/20/commits/c31f4975b32abec9e88b065933044bdb9cde1655) and add a file `.exclude-rhel7` into the versioned directory instead of renaming the Dockerfile.